From 863fbba11458cd95480e5e376bb42ade2969be39 Mon Sep 17 00:00:00 2001
From: Julius Kammerl <julius@kammerl.de>
Date: Mon, 25 Feb 2013 12:19:33 -0800
Subject: [PATCH 18/18] #pragma warning is MSVS preprocessor code

---
 common/include/pcl/point_types.h |    9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/common/include/pcl/point_types.h b/common/include/pcl/point_types.h
index ff3fbf0..6f54942 100644
--- a/common/include/pcl/point_types.h
+++ b/common/include/pcl/point_types.h
@@ -53,7 +53,10 @@
 
 // We're doing a lot of black magic with Boost here, so disable warnings in Maintainer mode, as we will never
 // be able to fix them anyway
-//#pragma warning(disable: 4201)
+#if defined _MSC_VER
+  #pragma warning(disable: 4201)
+#endif
+
 //#pragma warning(push, 1)
 #ifdef BUILD_Maintainer
 #  if defined __GNUC__
@@ -586,7 +589,9 @@ namespace pcl {
   };
 } // namespace pcl
 
-#pragma warning(default: 4201)
+#if defined _MSC_VER
+  #pragma warning(default: 4201)
+#endif
 //#pragma warning(pop)
 #ifdef BUILD_Maintainer
 #  if defined __GNUC__
-- 
1.7.9.5

